prac_06/programming_language.py

```diff
@@ -0,0 +1,26 @@
+"""
+CP1404/CP5632 Practical
+
+Predicted: 10 mins
+Actual: 20 mins
+
+"""
+
+
+class ProgrammingLanguage:
+    """Programming Language class, has name, typing, reflection, year"""
+
+    def __init__(self, name, typing, reflection, year):
```

Comment:
Consider defining the output with commenting . While clear for this example may be more useful in other scripts where the result is more ambiguous (whether its a str or boolean output)